examples/basics/export_data.ipynb

```diff
@@ -125,6 +125,13 @@
       "outputs": [],
```

Comment:
<p>I think this unecessary on each section, I think we should remove it but if you feel strongly about it, perhaps we can only included it at the end of the file. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>